docs/2023/Text2SQL/Performance, Cost Optimization, Benchmarking/updates/2023-07-14.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Week 2
+author: Pramod Goyal
+---
+
+## Milestones
+- The project had multiple dependency issues, that were resolved with the help of @suyashgautam
+- Work was done to add support for sqlite in the current project
+<!-- - [ ] Give the description about Milestone 3 -->
+<!-- - [ ] Give the description about Milestone 4 -->
+
+<!-- ## Screenshots / Videos  -->
+
+<!-- ## Contributions -->
+
+## Learnings
+- Postman 
```

Comment:
These can be a little more descriptive

Reply:
It could be, and should be. Done. Kindly let me know if there are any other changes you would like me to make.